sig-release/members.md

```diff
@@ -0,0 +1,8 @@
+## SIG Release Membership
```

Comment:
I think Google Group membership is insufficient for listing members for the purposes of communication. Personally communicating with me via email is the _least_ effective way to reach me and my Google+ profile does not link my GitHub id which is practically a much more important identifier for the project. Moreover I would like to have all of our documentation listed in source control somewhere.

There is also a thread about having GitHub team membership information stored in source control for greater visibility for the community and I thought it makes sense to support that idea here.